src/main/java/com/xiaomi/infra/pegasus/operator/client_operator.java

```diff
@@ -10,26 +10,22 @@
 import com.xiaomi.infra.pegasus.thrift.TException;
 
 public abstract class client_operator {
-  public client_operator(gpid gpid, String tableName, boolean enableBackupRequest) {
+
+  public client_operator(
+      gpid gpid, String tableName, long partitionHash, boolean enableBackupRequest) {
     this.header = new ThriftHeader();
     this.meta = new request_meta();
     this.meta.setApp_id(gpid.get_app_id());
     this.meta.setPartition_index(gpid.get_pidx());
+    this.meta.setPartition_hash(partitionHash);
     this.pid = gpid;
     this.tableName = tableName;
     this.rpc_error = new error_code();
     this.enableBackupRequest = enableBackupRequest;
```

Comment:
add 'this.enableSizeLimit = false' to imply the enableSizeLimit is fase by default

Reply:
Java `boolean` default value is `false`, so I don't add it explicitly